From: Jakub Jirutka <jakub@jirutka.cz>
Date: Sat, 18 Mar 2023 23:10:41 +0100
Subject: [PATCH] Log MBIM debug message on debug level

Don't pollute syslog with debug messages, unless running with --debug.

--- a/plugins/mbim.c
+++ b/plugins/mbim.c
@@ -61,7 +61,7 @@
 {
 	const char *prefix = user_data;
 
-	ofono_info("%s%s", prefix, str);
+	DBG("%s%s", prefix, str);
 }
 
 static int mbim_parse_descriptors(struct mbim_data *md, const char *file)
